### PR TITLE
update jit-related dependencies

### PR DESCRIPTION
**Pull request by xudong963**
# Which issue does this PR close?

Closes #1950 #1949 #1948 #1947

### CODE REVIEW

**Comment by xudong963** [comment]
cc @yjshen

File: datafusion-jit/src/jit.rs [hunk 1]
```diff
@@ -45,7 +45,7 @@ impl Default for JIT {
     #[cfg(target_arch = "x86_64")]
     fn default() -> Self {
         let builder = JITBuilder::new(cranelift_module::default_libcall_names());
```
Comment (yjshen) [inline datafusion-jit/src/jit.rs:47]: perhaps unwrap() on this line?

File: datafusion-jit/src/jit.rs [hunk 2]
```diff
@@ -99,7 +99,7 @@ impl JIT {
         });
         let isa = isa_builder.finish(settings::Flags::new(flag_builder));
```
Comment (yjshen) [inline datafusion-jit/src/jit.rs:100]: Same here.

**Review by yjshen — APPROVED** [review]
Thanks @xudong963 !

**Review by Jimexist — APPROVED** [review]
thank you

**Pull request merged**